deploy/vagrant/Vagrantfile

```diff
@@ -0,0 +1,56 @@
+$provisioner_ip_address = '10.11.12.2'
```

Comment:
Thoughts on using the IP range from setup.sh and the terraform on packet? I.e. 192.168.1.0/24

Reply:
Yep, it does this now.